bbb-install-2.5.sh

```diff
@@ -899,6 +899,24 @@ HERE
     chown bigbluebutton:bigbluebutton $TARGET
     chmod 644 $TARGET
   fi
+
+  # Override file for bbb-webrtc-sfu configs
+  BBB_WEBRTC_SFU_OVERRIDE_DIR=/etc/bigbluebutton/bbb-webrtc-sfu
+  BBB_WEBRTC_SFU_OVERRIDE_F=${BBB_WEBRTC_SFU_OVERRIDE_DIR}/production.yml
+
+  # Create SFU's override dir and file if they're not present yet
+  if [ ! -f "$BBB_WEBRTC_SFU_OVERRIDE_F" ]; then
+    mkdir -p "$BBB_WEBRTC_SFU_OVERRIDE_DIR"
+    touch "$BBB_WEBRTC_SFU_OVERRIDE_F"
```

Comment:
Permissions of the override file should be set. Otherwise permissions depend on the umask

```suggestion
    touch "$BBB_WEBRTC_SFU_OVERRIDE_F"
    chmod 600 "$BBB_WEBRTC_SFU_OVERRIDE_F"
    chown bigbluebutton:root "$BBB_WEBRTC_SFU_OVERRIDE_F"
```